src/System.Net.Http/src/System/Net/Http/Unix/CurlHandler.CurlResponseMessage.cs

```diff
@@ -426,10 +431,25 @@ private void CheckDisposed()
                 }
             }
 
-            [Conditional(VerboseDebuggingConditional)]
-            private void VerboseTrace(string text = null, [CallerMemberName] string memberName = null)
+            private void EventSourceTrace<T>(string formatMessage, T arg0, [CallerMemberName] string memberName = null)
+            {
+                if (EventSourceTracingEnabled)
+                {
+                    EventSourceTrace(string.Format(formatMessage, arg0), memberName);
+                }
+            }
+
+            private void EventSourceTrace<T1, T2, T3>(string formatMessage, T1 arg0, T2 arg1, T3 arg2, [CallerMemberName] string memberName = null)
```

Comment:
Nit: if this is going to stay a generic method, I think it would make sense to line be consistent in the zero- or one-based numbering of args and arg types.

Either T0 arg0, T1 arg1, ...
Or T1 arg1, T2 arg2...